cypress/integration/login.js

```diff
@@ -8,20 +8,87 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Check login as Admin and if the day is today', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Edvinas Kveselys"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Admin"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Edvinas Kveselys');
+        cy.get('.main-nav').find('li').should('have.length', 6);
+
+        const today = new Date();
+        const day = today.getDate();
+        cy.get('.calendar__date').contains(day);
+    })
+
+    it('Verify "User" can log in and should see appropriate tabs', function() {

```

Comment:
Name "can see appropriate tabs" imply that you check that he sees specific tabs, but you actually   only check the count